resources/migrations/5_v0.5.14.up.sql

```diff
@@ -0,0 +1,129 @@
+    ADD COLUMN "forwarded_values_query_string_cache_keys" _TEXT;
+ALTER TABLE "aws_cloudfront_distributions"
+    ADD COLUMN "max_ttl" INT8;
+ALTER TABLE "aws_cloudfront_distributions"
+    ADD COLUMN "min_ttl" INT8;
+ALTER TABLE "aws_cloudfront_distributions"
+    ADD COLUMN "origin_request_policy_id" TEXT;
+ALTER TABLE "aws_cloudfront_distributions"
+    ADD COLUMN "realtime_log_config_arn" TEXT;
+ALTER TABLE "aws_cloudfront_distributions"
+    ADD COLUMN "smooth_streaming" BOOL;
+ALTER TABLE "aws_cloudfront_distributions"
+    ADD COLUMN "trusted_key_groups_enabled" BOOL;
+ALTER TABLE "aws_cloudfront_distributions"
+    ADD COLUMN "trusted_key_groups" _TEXT;
+ALTER TABLE "aws_cloudfront_distributions"
+    ADD COLUMN "trusted_signers_enabled" BOOL;
+ALTER TABLE "aws_cloudfront_distributions"
+    ADD COLUMN "trusted_signers" _TEXT;
+
+--aws_cloudfront_distribution_cache_behavior_lambda_functions
+CREATE TABLE aws_cloudfront_distribution_cache_behavior_lambda_functions
+(
+    cq_id                             uuid NOT NULL,
+    meta                              jsonb NULL,
+    distribution_cache_behavior_cq_id uuid NULL,
+    event_type                        text NULL,
+    lambda_function_arn               text NULL,
+    include_body                      bool NULL,
+    CONSTRAINT aws_cloudfront_distribution_cache_behavior_lambda_functions_pk PRIMARY KEY (cq_id)
+);
+
+--aws_cloudfront_distribution_default_cache_behavior_lambda_functions
+ALTER TABLE "aws_cache_behaviour_lambda_function_associations" RENAME TO "aws_cloudfront_distribution_default_behaviour_lambda_functions";
+
+--aws_cloudfront_distribution_origin_groups
+ALTER TABLE "aws_cloudfront_distribution_origin_groups" RENAME COLUMN "failover_criteria_status_codes_items" TO "failover_criteria_status_codes";
+
+
+--aws_cloudtrail_trails
+ALTER TABLE "aws_cloudtrail_trails"
+    ADD COLUMN "tags" json;
+ALTER TABLE "aws_cloudtrail_trails"
+DROP
+COLUMN "home_region";
+
+--aws_elasticbeanstalk_environments
+ALTER TABLE "aws_elasticbeanstalk_environments"
+    ADD COLUMN "tags" json;
+ALTER TABLE "aws_elasticbeanstalk_environments" RENAME COLUMN "environment_name" TO "name";
+
+--aws_elasticsearch_domains
+ALTER TABLE "aws_elasticsearch_domains"
+    ADD COLUMN "tags" json;
+
+--aws_elbv2_load_balancers
+ALTER TABLE "aws_elbv2_load_balancers"
+    ADD COLUMN "tags" json;
+
+--aws_elbv2_target_groups
+ALTER TABLE "aws_elbv2_target_groups"
+    ADD COLUMN "tags" json;
+
+--aws_kms_keys
+ALTER TABLE "aws_kms_keys"
+    ADD COLUMN "tags" json;
+ALTER TABLE "aws_kms_keys" RENAME COLUMN "key_id" TO "id";
+
+--aws_elbv2_load_balancer_availability_zones
+ALTER TABLE "aws_elbv2_load_balancer_availability_zones"
+DROP
+CONSTRAINT "aws_elbv2_load_balancer_availability_zones_pk";
+
+--aws_elbv2_load_balancer_availability_zone_addresses
+ALTER TABLE "aws_elbv2_load_balancer_availability_zone_addresses"
+DROP
+CONSTRAINT "aws_elbv2_load_balancer_availability_zone_addresses_pk";
```

Comment:
if you are dropping a constraint (IF IT EXISTS) you need to create a new PK constraint on the cq_id